src/main/java/pl/isa/freshmenindustries/Main.java

```diff
@@ -27,15 +28,14 @@ public static void main(String[] args) {
 
                 if (haslo.equals(zarejestrowaniUzytkownicy.get(nazwaUzytkownika))) {
                     System.out.println("Witaj " + nazwaUzytkownika + ".");
-
                     BasicOptions basicOptions = new BasicOptions();
                     basicOptions.displayOptions();
                     basicOptions.enterNumber();
                     try {
-                        basicOptions.enterInput();
+                        basicOptions.enterInput(GameList.games());
```

Comment:
Skoro funkcja GameList.games() jest statyczna, to po co jest przekazywana jako argument funkcji już tutaj...